src/Admin/AbstractAdmin.php

```diff
@@ -2038,7 +2038,7 @@ public function transChoice(string $id, int $count, array $parameters = [], ?str
 
         $domain = $domain ?: $this->getTranslationDomain();
 
-        return $this->translator->transChoice($id, $count, $parameters, $domain, $locale);
+        return $this->trans($id, $parameters, $domain, $locale);
```

Comment:
Your changing the behaviour, isn't it ?

Since this method is deprecated, it's better to remove usage of it and remove this method.